packages/atomic/tsconfig.json

```diff
@@ -11,7 +11,17 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "jsx": "react",
-    "jsxFactory": "h"
+    "jsxFactory": "h",
+    "plugins": [
+      {
+        "name": "ts-lit-plugin",
```

Comment:
Does this need to be added to dev dependencies? It seems missing from dependencies, right now.

Reply:
No, not needed.

It's used by the cli tool `lit-analyzer`.